.github/workflows/build-and-test-types.yml

```diff
@@ -179,13 +189,20 @@ jobs:
         working-directory: ./redux-toolkit/examples/publish-ci/${{ matrix.example }}
         run: yarn info reselect && yarn why reselect
 
+      - name: Set up JDK 17 for React Native build
+        if: matrix.example == 'react-native'
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17.x'
+          distribution: 'temurin'
+
       - name: Check MSW version
         working-directory: ./redux-toolkit/examples/publish-ci/${{ matrix.example }}
         run: yarn why msw
 
       - name: Build example
         working-directory: ./redux-toolkit/examples/publish-ci/${{ matrix.example }}
-        run: yarn build
+        run: NODE_OPTIONS=--openssl-legacy-provider yarn build
```

Comment:
huh, how come this is needed?

Reply:
Not sure tbh, but I remember we had to add it to RTK as well to get the CI to work.

Reply:
hmm, okay